components/_quiz-card.css

```
@@ -1,3 +1,54 @@
 .quiz-card {
-  background-color: var();
+  border-radius: 15px;
+  padding: 25px;
+  margin: 20px;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+}
+
+.quiz-card__box {
+  background-color: white;
+  opacity: 0.5;
+  box-shadow: var(--box-shadow);
+}
+
+.quiz-card__heading {
+  font-size: 1.1em;
+}
+
+.quiz-card__bookmark--icon {
+  position: absolute;
+  height: 30px;
+  right: 30px;
+  top: -1px;
+}
+
+.quiz-card__btn {
+  background-color: var(--aqua);
+  border: none;
+  border-radius: 5px;
+  opacity: 0.8;
+  width: 100%;
+}
+
+.quiz-card__tag-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  flex-wrap:
+  margin: 0 0 3px 0;
+  padding: 0;
+}
+
+.quiz-card__tag-list--item {
```

`--something` indicates that this is a modifier of a style. For example `card__bookmark` and `card__bookmark--active`, the later is a modifier for the bookmark if you have clicked on it. The tag list item is not a modifier of your tag list.
```suggestion
.quiz-card__tag {
```